CHANGES.rst

```diff
@@ -446,6 +446,9 @@ astropy.time
 - Initialization of Time instances now is consistent for all formats to
   ensure that ``-0.5 <= jd2 < 0.5``. [#6653]
 
+- Initialization of ``Time`` instances with ``Column`` is now possible
```

Comment:
Sorry for missing this, but this should reference the actual change here of generically allowing byte-type input (either bytes or dtype='S' ndarray objects).
```
In [2]: Time(Column(b'2001:001'))
Out[2]: <Time object: scale='utc' format='yday' value=2001:001:00:00:00.000>

In [3]: Time(b'2001:001')
Out[3]: <Time object: scale='utc' format='yday' value=2001:001:00:00:00.000>

In [4]: Time(np.array([b'2001:001']))
Out[4]: <Time object: scale='utc' format='yday' value=['2001:001:00:00:00.000']>
```